llvm/test/ExecutionEngine/JITLink/AArch64/COFF_adrp.s

```diff
@@ -0,0 +1,32 @@
+# REQUIRES: asserts

```

Comment:
I think this would be better written as a `jitlink-check` test. I know you mentioned that you didn't have a Windows machine handy to test on, but it should still be possible to write `-noexec` tests on any platform (even non-aarch64 ones).